test/util/conf_test.py

```diff
@@ -21,6 +21,8 @@
     # You can specify arbitrary keys, but they'll get ignored if they aren't
     # actually declared anywhere (included here to make sure of that).
     '--config_value=undeclared_flag=who_cares',
+    '--config-value', 'true_value=true',
```

Comment:
can you make one of them '--config-value=true_value=true'?

Reply:
yups!